master/internal/config/resource_manager_config.go

```diff
@@ -97,8 +97,9 @@ type AgentResourceManagerConfig struct {
 	RequireAuthentication bool   `json:"require_authentication"`
 	ClientCA              string `json:"client_ca"`
 
-	Name     string            `json:"name"`
-	Metadata map[string]string `json:"metadata"`
+	Name        string            `json:"name"`
+	ClusterName string            `json:"cluster_name"`
```

Comment:
what is the difference between `Name` and `ClusterName`? IIRC, the resource manager name is interchangeable with the cluster name

Reply:
Yea! I have a [draft PR](https://github.com/determined-ai/determined/pull/9389) up right now to replace `name` with `cluster_name`, ideally landing that before we land Saloni's feature branch since it's a breaking change. Once that lands and this PR is rebased onto that, the `clusterName` / `cluster_name` related changes should no longer show up in the diff!